mender-convert-modify

```diff
@@ -102,17 +102,23 @@ mkdir -p work/deb-packages
 log_info "Installing Mender client and related files"
 
 deb_arch=$(probe_debian_arch_name)
+deb_distro=$(probe_debian_distro_name)
```

Comment:
Suggestion:
Remember this will not only be run on debian images, necessarily, even though it is the main intention.

Reply:
~Yes, that was the case before also. That is why we manually unpack the contents of the packages instead of using any `dpkg` -like tools.~

You made me realize that we need to check if distro is in supported distros (`debian`, `ubuntu`), otherwise default to Debian. Thanks!